src/System.CommandLine.Rendering/TestTerminal.cs

```diff
@@ -200,6 +207,8 @@ public virtual ConsoleColor ForegroundColor
             }
         }
 
+        public bool VirtualTerminalMode { get; set; } = true;
```

Comment:
Minor deal here, but the name `VirtualTerminalMode` does not really indicate a bool. Perhaps something like `IsVirtualTerminalModeEnabled`?